.github/workflows/zap_export.yml

```diff
@@ -88,39 +81,52 @@ jobs:
       #   run: |
       #     ./zap.sh upload_recoded_bq ${{ matrix.entity }} $VERSION
 
-      - name: Archive to data library - PGdump
-        env:
-          VERSION: ${{ steps.version.outputs.version }}
-        run: |
-          echo "exporting ${{ matrix.entity }}.sql"
-          docker run --rm \
-            -e AWS_S3_ENDPOINT=$AWS_S3_ENDPOINT\
-            -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID\
-            -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY\
-            -e AWS_S3_BUCKET=$AWS_S3_LIBRARY_BUCKET\
-            -v $(pwd)/templates:/templates\
-            -v $(pwd)/.output:/.output\
-            nycplanning/library:ubuntu-01d9e5eddf65dbdea3eb0c500314963b5ec6246a library archive -f /templates/${{ matrix.entity }}.yml -v $VERSION -s -l -o pgdump --compress
+      - uses: NYCPlanning/action-library-archive@v1.1
+        with:
+          name: ${{ matrix.entity }}
+          path: products/zap-opendata/templates/${{ matrix.entity }}.yml
+          version: $VERSION
+          s3: true
+          latest: true
+          compress: true
+          output_format: pgdump csv
 
-      - name: Archive to data library - CSV
-        env:
-          VERSION: ${{ steps.version.outputs.version }}
-        run: |
-          echo "exporting ${{ matrix.entity }}.csv"
-          docker run --rm \
-            -e AWS_S3_ENDPOINT=$AWS_S3_ENDPOINT\
-            -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID\
-            -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY\
-            -e AWS_S3_BUCKET=$AWS_S3_LIBRARY_BUCKET\
-            -v $(pwd)/templates:/templates\
-            -v $(pwd)/.output:/.output\
-            nycplanning/library:ubuntu-01d9e5eddf65dbdea3eb0c500314963b5ec6246a library archive -f /templates/${{ matrix.entity }}.yml -v $VERSION -s -l -o csv --compress
+      # - name: Archive to data library - PGdump
```

Comment:
Want to just drop these comments if it's working as desired now?

Reply:
yup! wanted to keep those details handy until it was working